tests/conftest.py

```diff
@@ -1211,6 +1214,37 @@ def sv_variant_objs(request, parsed_sv_variants, institute_obj):
     return (build_variant(variant, institute_obj) for variant in parsed_sv_variants)
 
 
+@pytest.fixture(scope="function")
+def fusion_clinical_file(request) -> str:
+    """Get the path to a variant file"""
+    return clinical_fusion_path
+
+
+@pytest.fixture(scope="function")
+def fusion_variants(request, fusion_clinical_file) -> VCF:
+    """Return a VCF object containing RNA fusion variants."""
+    return VCF(fusion_clinical_file)
+
+
+@pytest.fixture(scope="function")
+def parsed_fusion_variants(request, fusion_variants, case_obj) -> Iterable:
+    """Get a generator with parsed fusion variants"""
+    individual_positions = {}
+    for i, ind in enumerate(fusion_variants.samples):
+        individual_positions[ind] = i
+
+    return (
+        parse_variant(variant, case_obj, individual_positions=individual_positions)
+        for variant in fusion_variants
+    )
+
+
+@pytest.fixture(scope="function")
+def fusion_variant_obj(request, parsed_fusion_variants, institute_obj) -> dict:
```

Comment:
I've added a few fixtures that should also indirectly test the parsing of the fusion variants. Something is not working as expected still..